scripts/setup-proof.py

```diff
@@ -17,16 +18,16 @@ def proof_template_filenames():
 
 def read_proof_template(filename):
     directory = os.path.join(util.templates_root(), util.PROOF_TEMPLATES)
-    with open(os.path.join(directory, filename)) as data:
+    with open(os.path.join(directory, filename), encoding='utf-8') as data:
```

Comment:
just curious, why is this important?

Reply:
Important?  Why, because pylint says it is important.  See `pylint --help-msg=W1514`